local-process-quickstart.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+# Please install the below pre-requisites if using this on your local machine, or alternately, you can just use azure bash cloudshell for a seamless experience.
+#1. az
+#2. kubectl
+#3. curl
+#4. gunzip
+#5. tar
+#6. perl
+#7. cut
```

Comment:
Actually it could make sense to provide links for these ones as well. Some of these packages have very generic names (cut, tar, etc.) and it would be great if we can ease our users lives by providing them directly what they need to install.

Reply:
These are very basic commands that 99% of the times, the user would already have based on the which flavor of bash you are using. Also different people may download tools in a different way. Like I usually use homebrew for ease of use, but you would need homebrew installed on your machine for that. So not sure which kind of links to put here. I think we can skip providing links here.